### DISCUSSION TITLE
New milestone, Connect to database system

### DISCUSSION THREAD

**Discussion by sarasikarunanayaka** [ousl-groupE5-ExpiryInspector/ExpiryInspector#51] [Announcements]
Hello everyone, 

All  User interfaces connecting to the  to database system should be completed before the  October 31, 2024. Please follow the issues once you are assigned them.

@thidaskaveesha @Pradeepkumara95 @Sahani13 

Thank you.

**Comment by Pradeepkumara95** [comment]
Okay, lets get it done before the deadline.
